server/portal/libs/agave/operations.py

```diff
@@ -478,5 +479,9 @@ def preview(client, system, path, href, max_uses=3, lifetime=600):
         url = 'https://nbviewer.jupyter.org/urls/{tmp}'.format(tmp=tmp)
     else:
         file_type = 'other'
+        txt = text_preview(url)
+        logger.debug(content)
 
+    if txt:
```

Comment:
When I preview `.png` files there is an instance when the `txt` variable is never defined. I'm not sure if this is an issue with other file types yet, but it should be looked into.

Reply:
Previewing `.txt` files will cause the preview window to flatten with a horizontal scroll bar.
**Master**
<img width="778" alt="before" src="https://user-images.githubusercontent.com/29575979/102282768-ba18da80-3ef6-11eb-9e89-190df310961f.png">
**FP-371**
<img width="841" alt="after" src="https://user-images.githubusercontent.com/29575979/102282815-cef56e00-3ef6-11eb-83bb-fd84f122b683.png">